src/prebid.js

```diff
@@ -332,6 +334,38 @@ $$PREBID_GLOBAL$$.renderAd = function (doc, id) {
 
 };
 
+const renderOutstream = function(renderFn, adObject) {
+  // collapse DFP div
+  document.getElementById(adObject.adUnitCode).firstChild.style.display = 'none';
+
+  // collapse ad unit div
+  document.getElementById(adObject.adUnitCode).style.display = 'none';
+
+  // call the render function
+  adObject.adResponse.ad = adObject.adResponse.ads[0];
+  adObject.adResponse.ad.video = adObject.adResponse.ad.rtb.video;
+  renderFn({
+    tagId: adObject.adResponse.tag_id,
+    sizes: [adObject.getSize().split('x')],
+    targetId: adObject.adUnitCode, // target div id to render video
+    uuid: adObject.adResponse.uuid, // is this the correct UUID
+    adResponse: adObject.adResponse
+  });
+};
+
+function performRenderViaRenderer(doc, adObject) {
+  window.apntag = { debug: true };
```

Comment:
this logic is specific to appnexus, we should move it into the adapter or into a renderer interface type.